problems.txt

```diff
@@ -0,0 +1,96 @@
+2017-07-06T17:31:59.182Z pid:20915 worker:1     at Module.load (module.js:503:32)
+2017-07-06T17:31:59.182Z pid:20915 worker:1   errno: -2,
+2017-07-06T17:31:59.182Z pid:20915 worker:1   code: 'ENOENT',
+2017-07-06T17:31:59.182Z pid:20915 worker:1   syscall: 'open',
+2017-07-06T17:31:59.182Z pid:20915 worker:1   path: './config.json' }
+2017-07-06T17:31:59.182Z pid:20915 worker:1 [2017-07-06 10:31:59.177] [ERROR] knj_restclient_log -  VCAP_APPLICATION env variable not found, faild to connect to AvailabilityMonitoring service.
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.177] [INFO] knj_restclient_log - No any service is bound for this application.
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.179] [ERROR] knj_restclient_log - No ingress URL is set, please set "ingressURL" in your configuration json file
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - Failed to get /home/sam/w/sn/strong-supervisor/bin//package.json
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - NPM Packages discovered:
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->appmetrics: ^3.x
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->appmetrics-dash: ^3.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->async: ^2.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->debug: ^2.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->dotenv: ^1.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->ibmapm: ^1.0.8
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->lodash: ^4.14.1
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->posix-getopt: ^1.1.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-cluster-control: ^2.0.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-control-channel: ^2.0.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-log-transformer: ^1.0.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-npm-ls: ^1.0.0
+2017-07-06T17:31:59.186Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-statsd: ^2.0.0
+2017-07-06T17:31:59.186Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-trace: ^1.2.0
+2017-07-06T17:31:59.187Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-url-defaults: ^1.2.0
+2017-07-06T17:31:59.187Z pid:20915 worker:1 [2017-07-06 10:31:59.182] [ERROR] knj_restclient_log - No ingress URL is set, please set "ingressURL" in your configuration json file
```

Comment:
given that you don't have configuration, you can't run, and are never going to be able to re-run (unless you are polling the configuration file to see if becomes valid?). If you can't run, you should log a message to stderr, and deactivate yourself.